sdk/plugin/pb/backend.proto

```diff
@@ -703,6 +713,9 @@ service SystemView {
   // GenerateIdentityToken returns an identity token for the requesting plugin.
   rpc GenerateIdentityToken(GenerateIdentityTokenRequest) returns (GenerateIdentityTokenResponse);
 
+  // GetRotationInformation returns the time remaining in a rotation job for the requested credential.
+  rpc GetRotationInformation(RotationInfoRequest) returns (RotationInfoReply);
```

Comment:
Think this comment differed (is missing) from Ent but that can be fixed after the merge into Ent.